src/CheckTreePicker/test/CheckTreePickerSpec.js

```diff
@@ -639,4 +639,24 @@ describe('CheckTreePicker', () => {
     expect(onChangeSpy.firstCall.args[0]).to.include('Master');
     expect(onChangeSpy.secondCall.args[0]).to.include('tester1');
   });
+
+  it('Should remove all value when click clean button and value is unControlled', () => {
+    const ref = React.createRef();
+    const screen = render(
+      <CheckTreePicker ref={ref} defaultOpen data={data} defaultValue={['Master']} />
+    );
+
+    fireEvent.click(screen.getByLabelText('Clear'));
+    expect(ref.current.target.querySelector('.rs-picker-toggle-value')).to.equal(null);
+  });
+
+  it('Should persist value when click clean button and value is controlled', () => {
+    const ref = React.createRef();
+    const screen = render(<CheckTreePicker ref={ref} defaultOpen data={data} value={['Master']} />);
+
+    fireEvent.click(screen.getByLabelText('Clear'));
+    expect(
+      ref.current.target.querySelector('.rs-picker-toggle-value .rs-picker-value-item').textContent
+    ).to.equal('Master (All)');
```

Comment:
The DOM can be obtained without relying on `ref`.
```suggestion
    expect(screen.getByRole('combobox')).to.text('Master (All)1');
```